addons/project_mail_plugin/controllers/project_client.py

```diff
@@ -0,0 +1,46 @@
+
+from odoo import http
+from odoo.http import request
+
+
+class ProjectClient(http.Controller):
```

Comment:
Side note: I would have routes for project then routes for task, separated by a small comment section.  That way we classify by model :)